content/runbooks/alertmanager/AlertmanagerClusterCrashlooping.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Alertmanager Cluster Crashlooping
+weight: 20
+---
+
+# AlertmanagerClusterCrashlooping
+
+## Meaning
+
+Half or more of the Alertmanager instances within the same cluster are crashlooping.
+
+## Impact
+
+Alerts could be notified multiple time unless pods are crashing to fast and no alerts can be sent.
+
+## Diagnosis
```

Comment:
Adding a reference command for checking the events related to the pod in question would be useful, as they generally contain the reason for crashing

like: `kubectl get events --field-selector involvedObject.name=alertmanager-main-0`